packages/workflows/src/handlers/product/attach-shipping-profile-to-products.ts

```diff
@@ -43,3 +40,7 @@ export async function attachShippingProfileToProducts({
     )
   )
 }
+
+attachShippingProfileToProducts.aliases = {
+  products: "products",
```

Comment:
Same here. Maybe a more general question; have something changed for what should be defined as aliases here? Previously, we defined each key of the input object here.

Reply:
Since the data are aggregated , if their name already matches whats needed then they don't need to be aliased to what they are already named